index.html

```diff
@@ -223,40 +226,85 @@ <h1>hackSocIndore</h1>
+                            </div>
+                                <div id="harsh" class="collapse" style="margin-top: 2em">
+                                    <p>Name: Harsh Mandalgi</p>
+                                    <p>Github: <a href="https://github.com/harshmandalgi">@harshmandalgi</a></p>
+                                </div>
+                            <div class="col-sm-2">
+                                <a href="#sarthak" data-toggle="collapse">
+                                    <img src="img/devlopers/sarthak.jpg" class="rounded-circle person" alt="Sarthak" width="90" height="90">
+                                </a>
+                            </div>
+                                <div id="sarthak" class="collapse" style="margin-top: 2em">
+                                    <p>Name: Sarthak Srivastava</p>
+                                    <p>Github: <a href="https://github.com/sarthaksri98">@SarthakSri98</a></p>
+                                </div>
+                            <div class="col-sm-2">
+                                <a href="#vaibhav" data-toggle="collapse">
+                                    <img src="img/devlopers/vaibhav.jpg" class="rounded-circle person" alt="Vaibhav" width="90" height="90">
+                                </a>
+                            </div>
+                                <div id="vaibhav" class="collapse" style="margin-top: 2em">
+                                    <p>Name: Vaibhav Shrivastava</p>
+                                    <p>Github: <a href="https://github.com/mishuvs">@mishuvs</a></p>
+                                </div>
+                            <div class="col-sm-2">
+                                <a href="#harshit" data-toggle="collapse">
+                                    <img src="img/devlopers/harshit.png" class="rounded-circle person" alt="Harshit" width="90" height="90">
+                                </a>
+                            </div>
+                                <div id="harshit" class="collapse" style="margin-top: 2em">
+                                    <p>Name: Harshit Likhar</p>
+                                    <p>Github: <a href="https://github.com/harshitlikhar">@harshitlikhar</a></p>
+                                </div>
+                            <div class="col-sm-2">
+                                <a href="#hariom" data-toggle="collapse">
+                                    <img src="img/devlopers/hariom.jpg" class="rounded-circle person" alt="Hariom" width="90" height="90">
+                                </a>
+                            </div>
+                                <div id="hariom" class="collapse" style="margin-top: 2em">
+                                    <p>Name: Hariom Verma</p>
+                                    <p>Github: <a href="https://github.com/harry-hov">@harry-hov</a></p>
+                                </div>
+                        </div>
```

Comment:
Are you sure the indentation is uniform?

Reply:
@mishuvs I'll resolve the comments, once my midsem gets over. And will implement the same suggested by you